dev/archery/archery/crossbow/core.py

```diff
@@ -329,9 +329,9 @@ def remote(self):
             return self.repo.remotes[self.branch.upstream.remote_name]
         except (AttributeError, KeyError):
             raise CrossbowError(
-                'Cannot determine git remote for the Arrow repository to '
-                'clone or push to, try to push the `{}` branch first to have '
-                'a remote tracking counterpart.'.format(self.branch.name)
+                f"Cannot determine git remote for the Arrow repository to "
```

Comment:
`f` is not needed for the first and third lines

Reply:
Resolved in [2c7c552](https://github.com/apache/arrow/pull/45629/commits/2c7c552cf2119490867016bdce97578af1f1481b).